.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -252,13 +252,14 @@ jobs:
     # Build Catalyst-Runtime
     - name: Build Catalyst-Runtime
       run: |
+        # TODO: Update LIGHTNING_GIT_TAG to 'latest_version' after v0.34.0
         cmake -S runtime -B runtime-build -G Ninja \
               -DCMAKE_BUILD_TYPE=Release \
               -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=$GITHUB_WORKSPACE/runtime-build/lib \
               -DPYTHON_EXECUTABLE=$(which python${{ matrix.python_version }}) \
               -Dpybind11_DIR=$(python${{ matrix.python_version }} -c "import pybind11; print(pybind11.get_cmake_dir())") \
               -DENABLE_LIGHTNING_KOKKOS=ON \
-              -DLIGHTNING_GIT_TAG="latest_release" \
+              -DLIGHTNING_GIT_TAG="8101e42" \
```

Comment:
@dime10 this is the latest commit including all fixes. Are you happy with this or you prefer to tag the "master" branch for wheels?

Reply:
The specific commit is good, for master we have the plugin test matrix :)